index/scorch/segment/zap/build.go

```diff
@@ -552,8 +552,7 @@ func persistDocValues(memSegment *mem.Segment, w *CountHashWriter,
 			nextPosting, err2 := postingsItr.Next()
 			for err2 == nil && nextPosting != nil {
 				docNum := nextPosting.Number()
-				docTermMap[docNum] = append(docTermMap[docNum], []byte(next.Term)...)
-				docTermMap[docNum] = append(docTermMap[docNum], termSeparator)
+				docTermMap[docNum] = append(append(docTermMap[docNum], []byte(next.Term)...), termSeparator)
```

Comment:
curious if this is faster or cosmetic?

Reply:
> curious if this is faster or cosmetic?

logically supposed to be faster -- there's fewer docTermMap reads & writes.  But, pragmatically, i didn't see any perf #'s effect from bleve-blast experiments.